### ISSUE TITLE
weird python version

### ISSUE THREAD

**Issue by beckermr** [regro/cf-scripts#1590]
This PR was odd:

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3949#issuecomment-1382178725

**Comment by jakirkham** [comment]
Yeah seeing `graalpy` in a bunch of unexpected places. Started with arch migration ( https://github.com/regro/cf-scripts/issues/1591 ), but as noted in that issue it is not isolated to arch

**Comment by jakirkham** [comment]
Did PR ( https://github.com/regro/cf-scripts/pull/1594 ) fix this?

**Comment by beckermr** [comment]
Not sure.
